schema/check_generated_data.py

```diff
@@ -15,20 +15,20 @@
 import schema_validator
 from schema_files import ALL_TEST_TYPES
 
+logger = logging.Logger("Checking Test Data vs. Schemas LOGGER")
+logger.setLevel(logging.INFO)
```

Comment:
Suggestion: often these libraries have a way to set the logging level from an environment variable or from the CLI, so that you can do something like `DEBUG=* ./genData` or `./genData -vv`

Reply:
Yes, that's possible. I'll take a look.

Also, I've removed lots of excessive logging in the latest commits.

One more thing: when executing in parallel, the individual threads cannot have a logger in their objects because a logger "cannot be pickled". That's a small complication / limit to logging.

Reply:
I think don't pickle the logger; just make a new one for each thread